build.gradle

```diff
@@ -78,7 +78,8 @@ tasks.register('acceptance-tests', Test) {
     useJUnitPlatform{
         includeTags "acceptance"
     }
-    testLogging.showStandardStreams = true
+    // toggle to see the HTTP traces when running acceptance tests
+    testLogging.showStandardStreams = false
```

Comment:
Having these logs enabled was challenging readability at CI most of the time IMO